layouts/kind/talk.html

```diff
@@ -0,0 +1,3 @@
+<h3>Talk abstract</h3>
+<%= yield %>
+<a class="btn btn-default" href="/2016-berlin/schedule/">Back to schedule</a>
```

Comment:
Should there be a TODO for enabling 2017 versions of the speaker and talk pages?